static/js/popovers.js

```diff
@@ -585,7 +585,7 @@ exports.render_actions_remind_popover = function (element, id) {
         ).flatpickr({
             enableTime: true,
             clickOpens: false,
-            defaultDate: moment().format(),
+            defaultDate: formatISO(new Date()),
```

Comment:
Based on the [flatpickr documentation](https://flatpickr.js.org/examples/#supplying-dates-for-flatpickr), this should just be `defaultDate: "today"`.